core/src/test/scala/unit/kafka/server/OffsetFetchRequestTest.scala

```diff
@@ -14,233 +14,532 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package kafka.server
```

Comment:
I rewrote all this test suite. The new one covers all the old cases as well, I think.